packages/wordpress-kit/src/lib/tailwindcssPlugin.ts

```diff
@@ -0,0 +1,113 @@
+import plugin from 'tailwindcss/plugin';
+
+const mergeToConfig = {
+  content: [],
+  safelist: [
+    '.has-primary-color',
+    '.has-secondary-color',
+    '.has-dark-gray-color',
+    '.has-white-color',
+
+    '.has-background',
+    '.has-primary-background-color',
+    '.has-secondary-background-color',
+    '.has-dark-gray-background-color',
+    '.has-white-background-color',
+
+    '.has-huge-font-size',
+    '.has-large-font-size',
+    '.has-normal-font-size',
+    '.has-small-font-size',
+    '.has-light-gray-color',
+
+    '.has-drop-cap',
+  ],
+};
+
+export default plugin(function ({ addUtilities, theme }) {
+  const fontSizeUtilties = {
+    '.has-huge-font-size': {
+      fontSize: `${theme('fontSize.7xl', '5rem')} !important`,
+    },
+    '.has-large-font-size': {
+      fontSize: `${theme('fontSize.4xl', '4rem')} !important`,
+    },
+    '.has-normal-font-size': {
+      fontSize: `${theme('fontSize.xl', '1.25rem')} !important`,
+    },
+    '.has-small-font-size': {
+      fontSize: `${theme('fontSize.sm', '1rem')} !important`,
+    },
+  };
+
+  const colorUtilties = {
```

Comment:
```suggestion
  const colorUtilities = {
```
Check spelling for Utilities
also for  
fontSizeUtilities,
backgroundUtilities,
dropCapUtilities,